src/sestest/ServerOptionBuilder.cs

```diff
@@ -16,38 +14,29 @@ public class ServerOptionBuilder
         private readonly ServerCommandLine _commandLine;
 
         // Reference to a store in which we can search for certificates
-        private readonly CertificateStore _certificateStore = new CertificateStore();
-
-        // Options used to configure validation
-        private readonly ServerOptionBuilderOptions _options;
+        private readonly ICertificateStore _certificateStore;
 
         /// <summary>
         /// Build an instance of <see cref="ServerOptions"/> from the information supplied on the 
         /// command line by the user
         /// </summary>
         /// <param name="commandLine">Command line parameters supplied by the user.</param>
-        /// <param name="options">Options for configuring validation.</param>
         /// <returns>Either a usable (and completely valid) <see cref="ServerOptions"/> or a set 
         /// of errors.</returns>
-        public static Errorable<ServerOptions> Build(
-            ServerCommandLine commandLine,
-            ServerOptionBuilderOptions options = ServerOptionBuilderOptions.None)
+        public static Errorable<ServerOptions> Build(ServerCommandLine commandLine, ICertificateStore certificateStore)
```

Comment:
I think we should get rid of the static method - it was there to hide `ServerOptionBuilderOptions.None`